RileyLinkKit/Extensions/NSDate.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  NSDate.swift
+//  RileyLink
+//
+//  Created by Pete Schwamb on 5/18/16.
+//  Copyright © 2016 Pete Schwamb. All rights reserved.
+//
+
+import Foundation
+
+public func ==(lhs: NSDate, rhs: NSDate) -> Bool {
```

Comment:
There's a third scope (default) which is `internal`. It's availabile only to the framework.